5_databases/week2/task1_2.sql

```diff
@@ -0,0 +1,74 @@
+USE hyf_lesson1;
+
+--1. Add a task  with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO task (title, description, created, updated, due_date, status_id, user_id)
+VALUES ('Be good', 'Stop beating your husband', '2023-11-16 12:00:00', '2023-11-16 12:00:00', '2023-12-01 12:00:00', 1, 3);
+
+--2. Change the title of a task
+UPDATE task
+set title = "Sleep"
+where title = "Wash clothes";
+
+
+SELECT *
+from task;
+
+--3. Change a task due date
+UPDATE task
+SET due_date = 2017-10-15 
+WHERE title = 'Wash clothes';
+
+--4. Change a task status
+UPDATE status
+SET name = 'Yet to begin'
+WHERE name = 'Not started';
+
+SELECT *
+from status;
+
+--5. Mark a task as complete
+UPDATE task
+SET status_id = 3
+WHERE title = 'Wash clothes';
+
+--6. Delete a task
+DELETE FROM task
```

Comment:
Use id so you dont accidently delete rows

Reply:
Updated as suggested!